scripts/issue/365/register_vaults.py

```diff
@@ -189,8 +189,193 @@
+    'v1',
+    'name=wBTC/Badger,protocol=Sushiswap,behavior=None',
+    3
+  ],
+  [
+    '0x88128580ACdD9c04Ce47AFcE196875747bF2A9f6',
+    'v1',
+    'name=wBTC/Digg,protocol=Sushiswap,behavior=None',
+    3
+  ],
+  [
+    '0x7e7E112A68d8D2E221E11047a72fFC1065c38e1a',
+    'v1',
+    'name=Digg,protocol=Badger,behavior=None',
+    3
+  ],
+  [
+    '0x19D97D8fA813EE2f51aD4B4e04EA08bAf4DFfC28',
+    'v1',
+    'name=Badger,protocol=Badger,behavior=None',
+    0
+  ],
+  [
+    '0x4b92d19c11435614CD49Af1b589001b7c08cD4D5',
+    'v1',
+    'name=wBTC,protocol=Yearn,behavior=None',
+    3
+  ],
+  [
+    '0x8a8FFec8f4A0C8c9585Da95D9D97e8Cd6de273DE',
+    'v1',
+    'name=wBTC/ibBTC,protocol=Sushiswap,behavior=None',
+    0
+  ],
+  [
+    '0x8c76970747afd5398e958bDfadA4cf0B9FcA16c4',
+    'v1',
+    'name=hBTC,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x55912D0Cf83B75c492E761932ABc4DB4a5CB1b17',
+    'v1',
+    'name=pBTC,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0xf349c0faA80fC1870306Ac093f75934078e28991',
+    'v1',
+    'name=oBTC,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x5Dce29e92b1b939F8E8C60DcF15BDE82A85be4a9',
+    'v1',
+    'name=bBTC,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0xBE08Ef12e4a553666291E9fFC24fCCFd354F2Dd2',
+    'v1',
+    'name=Tricrypto,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x2B5455aac8d64C14786c3a29858E43b5945819C0',
+    'v1',
+    'name=cvxCRV,protocol=Convex,behavior=Ecosystem Helper',
+    3
+  ],
+  [
+    '0x53C8E199eb2Cb7c01543C137078a038937a68E40',
+    'v1',
+    'name=CVX,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x27E98fC7d05f54E544d16F58C194C2D7ba71e3B5',
+    'v1',
+    'name=Tricrypto2,protocol=Convex,behavior=None',
+    3
+  ],
+  [
+    '0x599D92B453C010b1050d31C364f6ee17E819f193',
+    'v1',
+    'name=imBTC,protocol=mStable,behavior=None',
+    0
+  ],
+  [
+    '0x26B8efa69603537AC8ab55768b6740b67664D518',
+    'v1',
+    'name=mhBTC,protocol=mStable,behavior=None',
+    0
+  ],
+  [
+    '0xfd05D3C7fe2924020620A8bE4961bBaA747e6305',
+    'v1',
+    'name=bveCVX,protocol=Convex,behavior=None',
+    3
+  ],
+  [
+    '0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551',
+    'v1',
+    'name=CVX/bveCVX,protocol=Curve,behavior=None',
+    3
+  ],
+  [
+    '0xaE96fF08771a109dc6650a1BdCa62F2d558E40af',
+    'v1',
+    'name=ibBTC/crvsBTC,protocol=Convex,behavior=None',
+    3
+  ],
+  [
+    '0x19E4d89e0cB807ea21B8CEF02df5eAA99A110dA5',
+    'v1',
+    'name=MIM/3CRV,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x15cBC4ac1e81c97667780fE6DAdeDd04a6EEB47B',
+    'v1',
+    'name=FRAX/3CRV,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x6aF7377b5009d7d154F36FE9e235aE1DA27Aea22',
+    'v1',
+    'name=remBadger,protocol=Badger,behavior=None',
+    3
+  ],
+  [
+    '0xeC1c717A3b02582A4Aa2275260C583095536b613',
+    'v1',
+    'name=BADGER/WBTC,protocol=Convex,behavior=None',
+    0
```

Comment:
why is status code here `0`?

Reply:
thanks, this was because it was 'featured' i've updated the gen script example for this.